src/main/res/menu/export_filter.xml

```diff
@@ -1,20 +0,0 @@
-<!--suppress ALL -->
```

Comment:
Moved into the nav drawer menu. I could have kept it separate and referenced it, but I don't think we'll ever use filter import/export anywhere else, so I'd rather have related items together.